src/battle-dex.ts

```diff
@@ -186,11 +186,16 @@ const Dex = new class implements ModdedDex {
 	})();
 
 	fxPrefix = (() => {
+		/*
 		if (window.document?.location?.protocol === 'file:') {
 			if (window.Replays) return `https://${window.Config ? Config.routes.client : 'play.pokemonshowdown.com'}/fx/`;
 			return `fx/`;
 		}
 		return `//${window.Config ? Config.routes.client : 'play.pokemonshowdown.com'}/fx/`;
+		*/
```

Comment:
```suggestion
```

I guess I never wrote this in CONTRIBUTING, but it's standard not to comment out code except in special cases.